test-app/package.json

```diff
@@ -23,8 +23,14 @@
     "test:ember": "ember test"
   },
   "dependencies": {
+    "ember-table": "^5.0.1",
     "embroider-bug": "workspace:*"
   },
+  "dependenciesMeta": {
```

Comment:
only done because I added ember-source to the peers list -- but this had no effect on the error, because the ultimate problem is that ember-table is ancient.